dataflow/src/ops/latest.rs

```diff
@@ -94,8 +94,8 @@ impl Ingredient for Latest {
             });
 
             // buffer emitted records
-            for (r, current) in currents {
-                if let Some(current) = current {
+            for (r, current_row) in currents {
+                if let Some(current) = current_row.get(0) {
                     out.push(Record::Negative((**current).clone()));
```

Comment:
Can we get rid of this clone (or one further down) if `current_row` is a `Cow::Owned`?

Reply:
Fixed here: https://github.com/mit-pdos/distributary/pull/72/commits/3a4a7c235f1e17dc8ec1084785be25a37e539a7c#diff-4822603b49efd860fa34b9ae63ccf3bdR99